package.json

```diff
@@ -10,7 +10,7 @@
         "url": "https://github.com/aws/aws-toolkit-vscode"
     },
     "engines": {
-        "vscode": "^1.31.1"
+        "vscode": "^1.43.0"
```

Comment:
What does this gain us?

Reply:
This is for the DAP provider + vscode-debugadapter dependency.

Reply:
https://github.com/aws/aws-toolkit-vscode/pull/1042